minidump-stackwalk/http_symbol_supplier.cc

```diff
@@ -123,18 +123,32 @@ HTTPSymbolSupplier::StoreSymbolStats(const CodeModule* module,
   }
 }
 
+string
+HTTPSymbolSupplier::GuessURL(const CodeModule* module)
+{
+  string path, url;
+  if (!server_urls_.empty()
+      && GetRelativeURLAndPathToSymbolFile(module, url, path)) {
+    return server_urls_.front() + url;
+  }
+  return "";
+}
+
 void
 HTTPSymbolSupplier::StoreCacheHit(const CodeModule* module)
 {
-  SymbolStats stats = {true, 0.0f};
+  // We don't know the original URL, so guess that it would have
+  // been from the first URL we would have tried.
```

Comment:
It would be nice if we didn't have to guess here, I suppose it'd need to keep state about where it downloaded files (or have a separate cache area for different source URLs).

I think this is fine for now but something to consider if we end up using multiple symbol URLs much - I don't expect that we will in the near term.